data_fusion/plugins/modules/procflows/data_fusion.py

```diff
@@ -529,6 +529,11 @@ def call(fnames, command_line_args=None):
         command_line_args, {"METADATA": fuse_data["final"]["metadata_xobj"]}
     )
 
+    if command_line_args.get("image_compare_threshold") is not None:
+        image_compare_threshold = command_line_args["image_compare_threshold"]
+    else:
+        image_compare_threshold = None
+
```

Comment:
I noticed when working on this PR, arguments "logging_level" and "no_presectoring" are not checked for. We might want to create an issue for these to be added in the future, and make the appropriate changes to the code.

Reply:
https://github.com/NRLMMD-GEOIPS/data_fusion/issues/26
Created an issue.  Resolving this!